www/FAQ.rst

```diff
@@ -41,7 +41,7 @@ really dislike C++.  For many reasons you can have hope that Swift
 will end up being a great "successor to Objective-C" instead of a "C++
 replacement":
 
- * The compiler team has expert-level knowledge of Objective-C (the language),
+ * The compiler team has an expert-level knowledge of Objective-C (the language),
```

Comment:
'Knowledge' is an uncountable noun; you cannot have a single knowledge. The original was correct.